azure-kusto-ingest/tests/e2e.py

```diff
@@ -58,9 +54,6 @@ class TestE2E:
     client: ClassVar[KustoClient]
     test_table: ClassVar[str]
     current_count: ClassVar[int]
-    ingest_client: ClassVar[QueuedIngestClient]
```

Comment:
Why did you remove these?

Reply:
mistake - will fix

Reply:
On second thought, any reason we initialize ingest_client + streaming_ingest_client + managed_streaming_ingest and not the ..._file_path variables declared in the setup_up func? In both instances these variables are only used in instance methods and not in class methods.. Whereas all other class initialized variables are used in class methods and must be declared

Reply:
I don't understand, we currently initialize both of them in the setup function for the class.
We don't have a setup function for an instance, though that's another topic.
Either way, we would want to declare them, if they were instance variables we would just do it without the ClassVar, the reason is for better autocompletion and error reporting.